src/index.html

```diff
@@ -15,8 +15,36 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo__img"><img src="./images/logo.png" alt="Moyo Logo"></a>
+      <nav class="navigation">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+
+            <a href="#samsung" class="nav__link">Samsung</a>
+
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+
+            <a href="#laptops&computers" class="nav__link"
+              data-qa="hover">Laptops & Computers</a>
```

Comment:
The line of code where `href` attribute is set exceeds the maximum of 80 characters. Consider breaking the line for better readability.